fe/fe-core/src/main/java/org/apache/doris/datasource/hive/HiveMetaStoreCache.java

```diff
@@ -296,8 +296,6 @@ public ListPartitionItem toListPartitionItem(String partitionName, List<Type> ty
         try {
             PartitionKey key = PartitionKey.createListPartitionKeyWithTypes(values, types, true);
             ListPartitionItem listPartitionItem = new ListPartitionItem(Lists.newArrayList(key));
-            // if `PartitionKey` is default, set `PartitionItem` to default
```

Comment:
`values.add(new PartitionValue(partitionValue, HIVE_DEFAULT_PARTITION.equals(partitionValue))`also need to modify？